content/ru/horizontal-scaling.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Горизонтальное масштабирование
+status: Completed
+category: Concept
+tags: ["infrastructure", "", ""]
+---
+
+Горизонтальное масштабирование — это метод, при котором производительность системы увеличивается за счет добавления [узлов](/nodes/). 
+Этим оно отличается от [вертикального масштабирования](/vertical-scaling/), когда ресурсы добавляются на сами узлы. 
+Допустим, у нас есть система с 4 ГБ ОЗУ и нужно увеличить ее емкость до 16 ГБ ОЗУ. 
+При горизонтальном масштабировании мы добавим еще три инстанса (узла) с 4 ГБ ОЗУ, при вертикальном увеличим объем ОЗУ до 16 ГБ на имеющемся инстансе.
+
+Такой подход позволяет повысить производительность приложения, добавляя новые инстансы или узлы, 
+чтобы лучше распределить рабочую нагрузку. 
+Проще говоря, он направлен на снижение нагрузки на сервер 
+а не на увеличение мощности отдельного сервера.
+
+## Какую проблему решает 
+
+По мере роста нагрузки на приложение возникает момент, когда она превышает возможности данного экземпляра приложения. 
+Чтобы система продолжала стабильно работать, необходимо ее [масштабировать](/scalability/) (увеличить ее ресурсы). 
+Для этого можно либо добавить больше узлов в систему (горизонтальное масштабирование), 
+либо увеличить объем вычислительных ресурсов на существующих узлах (вертикальное масштабирование).
+
+## Как именно решает проблему
+
+Горизонтальное масштабирование позволяет приложениям масштабироваться до пределов всего кластера. 
+Добавляя в систему больше экземпляров, мы увеличиваем число запросов, которое может обработать приложение. 
+Например, если один узел обрабатывает 1 тыс. запросов в секунду, 
+то два узла будут обрабатывать 2 тыс., три узла — 3 тыс. запросов (при условии, что узлы одинаковые) и т.д. 
```

Comment:
It's not mentioned in the English version and seems to be obvious (keeping the text easier to read).
```suggestion
то два узла будут обрабатывать 2 тыс., три узла — 3 тыс. запросов и т.д. 
```